### ISSUE TITLE
question for code

### ISSUE THREAD

**Issue by SuiYm** [MertCokelek/SalViT360#4]
Hi @MertCokelek,
Thanks for your excellent work! I have a couple of questions I'd like to ask you:
First, regarding lines 117-119 in trainer.py:

                        loss_kl, loss_cc, loss_nss = self.criterion(pred_erp, y_erp, self.overlap_mask)
                        loss_nss = loss_nss * self.w_nss
                        loss = self.w_kl * loss_kl + self.w_cc * loss_cc + self.w_nss * loss_nss

It seems like loss_nss is multiplied by self.w_nss twice here. Should this be corrected to:loss = self.w_kl * loss_kl + self.w_cc * loss_cc + loss_nss?
Second, your paper mentions that α = 0.005, but in the training configuration, it states that the best weight is 0.05. Could you clarify which one is the correct best weight?
